scripts/lw-scan-images.sh

```diff
@@ -26,30 +26,45 @@ if [ ! -f "$SCANNER" ]; then
   chmod +x "$SCANNER"
 fi
 
-echo "Gathering list of _all_ images for $VERSION"
+OCI_TOOL="$BIN/oci-tool"
```

Comment:
Maybe checking if `oci-tool` exists, like
```
if ! command -v oci-tool &> /dev/null; then
  curl -fsSL https://github.com/csweichel/oci-tool/releases/download/v${OCI_TOOL_VERSION}/oci-tool_${OCI_TOOL_VERSION}_linux_amd64.tar.gz | sudo tar xz -C /usr/local/bin
  sudo chmod +x /usr/local/bin/oci-tool
fi
```

Reply:
that can be done in a different PR too

Reply:
Ultimately it would be nice if we had the `oci-tool` pre-installed everywhere.
Until then I like the explicit version reference and the predictability it brings.

Reply:
Is already installed in the docker image we use in CI 😉

Reply:
https://github.com/gitpod-io/gitpod/blob/33eb48f44d633f9387806241bc7951780dd588c9/.github/workflows/lacework-inline-scanner.yml#L52 is what's used here.
I assumed we use the same image in workspace and CI (at least semantically), and when running in my workspace, `oci-tool` was not there.

Reply:
I am sorry. I assumed we already merged https://github.com/gitpod-io/gitpod/pull/18309/files